packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -118,6 +118,46 @@ void main() {
     expect(tester.testTextInput.setClientArgs!['inputAction'], equals(serializedActionName));
   }
 
+  testWidgets('when open input connection in didUpdateWidget, should not throw', (WidgetTester tester) async {
+    final Key key = GlobalKey();
+
+    await tester.pumpWidget(
+      MaterialApp(
+        home: EditableText(
+          key: key,
+          backgroundCursorColor: Colors.grey,
+          controller: TextEditingController(text: 'blah blah'),
+          focusNode: focusNode,
+          readOnly: true,
+          style: textStyle,
+          cursorColor: cursorColor,
+          selectionControls: materialTextSelectionControls,
+        ),
+      ),
+    );
+
+    focusNode.requestFocus();
+    await tester.pump();
+
+    // This pump will throw before https://github.com/flutter/flutter/pull/126324 is fixed
```

Comment:
I would add a comment above this test like this:

```
// Regression test for https://github.com/flutter/flutter/issues/126312.
```

Then change this comment to:

```
// Reparent the EditableText, so that the parent has not yet been laid
// out when didUpdateWidget is called.
```

Reply:
done